packages/core/src/common/_variables.scss

```diff
@@ -117,11 +117,8 @@ $pt-transition-duration: 100ms !default;
 
 // Light theme styles
 
-$pt-input-box-shadow: inset 0 1px 1px rgba($black, 0.5),
-                      inset 0 1px 0 rgba($black, 0.2),
-                      inset -1px 0 0 rgba($black, 0.2),
-                      inset 0 -1px 0 rgba($black, 0.2),
-                      inset 1px 0 0 rgba($black, 0.2) !default;
+$pt-input-box-shadow: inset 0 0 0 1px rgba(17, 20, 24, 0.2),
+                      inset 0 1px 1px rgba(17, 20, 24, 0.5) !default;
```

Comment:
actually these should stay as `$black` references

Reply:
ooh good catch - pushing up the fix now